mcux/hal_nxp.cmake

```diff
@@ -217,6 +217,8 @@ if (CONFIG_BOARD_MIMXRT1050_EVK_QSPI)
 endif()
 elseif(CONFIG_BOARD_MIMXRT1060_EVK OR CONFIG_BOARD_MIMXRT1060_EVK_HYPERFLASH)
   set(MCUX_BOARD evkmimxrt1060)
+elseif(CONFIG_BOARD_MIMXRT1060_EVKB)
```

Comment:
Can we add this to the previous case as an OR

Reply:
We need to select the evk*b*mimxrt1060 (b-variant) so we can't add it to the previous elif